extensions/wikia/CommunityPage/models/CommunityPageSpecialInsightsModel.class.php

```diff
@@ -156,22 +153,22 @@ private function getArticleContributeText( $metadata, $type ) {
 	private function getArticleMetadataDetails( $metadata ) {
 		global $wgUser, $wgLang;
 
-		if ( !empty( $metadata['wantedBy'] ) ) {
-			return wfMessage( $metadata['wantedBy']['message'] )->rawParams(
+		if ( !empty( $metadata[ 'wantedBy' ] ) ) {
+			return wfMessage( $metadata[ 'wantedBy' ][ 'message' ] )->rawParams(
 				Html::element(
 					'a',
 					[
-						'href' => $metadata['wantedBy']['url'],
+						'href' => $metadata[ 'wantedBy' ][ 'url' ],
 						'data-tracking' => 'wanted-by-link',
 					],
-					$wgLang->formatNum( $metadata['wantedBy']['value'] )
+					$wgLang->formatNum( $metadata[ 'wantedBy' ][ 'value' ] )
 				)
 			)->escaped();
 		}
 
-		$timestamp = wfTimestamp( TS_UNIX, $metadata['lastRevision']['timestamp'] );
+		$timestamp = wfTimestamp( TS_UNIX, $metadata[ 'lastRevision' ][ 'timestamp' ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "lastRevision" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)